draft-irtf-cfrg-frost.md

```diff
@@ -920,8 +920,8 @@ The value of the contextString parameter is "FROST-RISTRETTO255-SHA512-v5".
 - Group: ristretto255 {{!RISTRETTO=I-D.irtf-cfrg-ristretto255-decaf448}}
   - Order: 2^252 + 27742317777372353535851937790883648493 (see {{RISTRETTO}})
   - Identity: As defined in {{RISTRETTO}}.
-  - RandomScalar: Implemented by generating a random 32-byte string and invoking
-    DeserializeScalar on the result.
+  - RandomScalar: Implemented by repeatedly generating a random 32-byte string and
+    invoking DeserializeScalar on the result until success.
```

Comment:
That particular wording can be a bit inefficient in some cases, e.g. here in ristretto255 approximately 7 out of 8 tries will fail because it's looking for a 253-bit scalar in a 256-bit buffer. We'd need to clear the upper bits. But it's becoming more and more awkward 😅 Maybe this warrants a new helper function?

Reply:
I don't think that's necessary, but I'm happy to look at PRs.